### DISCUSSION TITLE
Introduce a Data Retention Policy

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#2147] [Anything Else]
By default we have unlimited data retention and people keep having access to the stats from the day they started using Plausible as long as they are active subscribers. it would be nice to introduce a Data Retention Policy for those that prefer to automatically delete their data after X days.

(request via email)

**Comment by simon-guider** [comment]
Working for a b2b company it would be useful to filter for props and delete all records with a specific prop
